owasp-top10-2017-apps/a5/ecommerce-api/app/handlers/handlers.go

```diff
@@ -13,14 +14,40 @@ func HealthCheck(c echo.Context) error {
 	return c.String(http.StatusOK, "WORKING\n")
 }
 
+func ErrorMessage(c echo.Context) error {
+	return c.JSON(http.StatusBadRequest, map[string]string{"result": "error", "details": "Error finding this UserID."})
+}
+
 // GetTicket returns the userID ticket.
 func GetTicket(c echo.Context) error {
+
+	cookie, err := ReadCookie(c)
+
+	if err != nil {
+		return ErrorMessage(c)
+	}
+
+	type UserClaims struct {
+		Name string `json:"name"`
+		jwt.StandardClaims
+	}
+
+	token, err := jwt.ParseWithClaims(cookie, &UserClaims{}, func(token *jwt.Token) (interface{}, error) {
+		return []byte("secret"), nil
+	})
+
+	claims, _ := token.Claims.(*UserClaims)
```

Comment:
Although this is not the case (as this second variable being "ignored" `_` is a boolean), keep in mind that using this to not check errors is a vulnerability in your code. 

Ref: G104: Audit errors not checked from [Gosec](https://github.com/securego/gosec)